src/api/include/migraphx/migraphx.hpp

```diff
@@ -34,6 +34,7 @@
 #include <exception>
 #include <array>
 #include <vector>
+#include <string_view>
```

Comment:
This cant use `string_view`. This is the API header and should be compatible with older C++ versions that users might be using.